apps/zbugs/server/server-mutators.ts

```diff
@@ -32,79 +33,75 @@ export function createServerMutators(
           created: Date.now(),
           modified: Date.now(),
         });
-        await notify(
-          tx,
-          authData,
-          {kind: 'create-issue', issueID: id},
-          postCommitTasks,
-        );
+        if (tx.location === 'server') {
```

Comment:
Why is this necessary? i thought this file was only used on server (hence name).

Is it basically casting `tx`? If so maybe this instead:

```
assert(tx.location === 'server');
```

Reply:
Yes it's only used on the server - it's a type helper to remove a downstream cast that was marked TODO. Changing to use asserts instead